Classes/Command/CreateTestDataCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Tea\Command;
+
+/*
+ * Command to create test data for the tea extension.
+ */
+
+use phpDocumentor\Reflection\Types\Integer;
```

Comment:
I assume that this is wrong - what's the reason to use the phpDocumentor reflection type here ? Maybe some error with auto import ?

Reply:
Yes, now removed.